community_tasks/arabic_evals.py

```diff
@@ -74,6 +74,46 @@ def mmlu_arabic(line, task_name: str = None):
         target_for_fewshot_sorting=LETTER_INDICES_AR[gold_ix],
     )
 
+# mbzuai_arabic_mmlu
+mbzuai_arabic_mmlu_task = LightevalTaskConfig(
+    name="mbzuai_arabic_mmlu",
+    prompt_function="mbzuai_arabic_mmlu",
```

Comment:
After recent PRs, this will need to be a callable function instead of a string.
```suggestion
    prompt_function=mbzuai_arabic_mmlu,
```

Reply:
Done. Check, please.